services/ui-src/src/components/GridWithTotals/GridWithTotals.js

```diff
@@ -16,6 +16,14 @@ const GridWithTotals = props => {
     updateTotals();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const roundValue = value => {
```

Comment:
It checks to see if the value contains a decimal or not. If it does not contain one, then don't display a decimal. If we don't use this and round all values that are returned, then every box contains a decimal. If we don't try to round all values that are returned, we need to check every value to see if it needs to be rounded or not.

Reply:
The cycle-minded guy in me wants to do this:
```

    if (value % 1 !== 0) {
      return value.toFixed(1);
    }
    return value;
```
This works in the console, and should be more efficient. Neither one behaves as desired with something like 1.99 (which, of course, rounds to 2.0) as a value, but I don't know if you're getting 2 digit decimals or not.

This may seem nitpicky, but this is easier to read and understand than what you had before, and doesn't introduce variables you don't need. It's very obvious what it's doing.

Reply:
This looks like a good simple refactor!